.env.example

```
@@ -92,6 +92,7 @@ NEXT_PUBLIC_CALENDLY_URL=
 API_PORT=3010
 API_SERVICE_NAME=api-genfeed-ai
 API_SENTRY_DSN=
+API_SENTRY_PERFORMANCE_METRICS=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_SENTRY_PERFORMANCE_METRICS key should go before the API_SERVICE_NAME key

(UnorderedKey)